### PR TITLE
spanish translation of challenge text

### PR DESCRIPTION
**Pull request by s-Hale**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Translated challenge text in "Make Screen Reader Navigation Easier with the Footer Landmark" into Spanish.

### CODE REVIEW

File: curriculum/challenges/spanish/01-responsive-web-design/applied-accessibility/make-screen-reader-navigation-easier-with-the-footer-landmark.spanish.md [hunk 1]
```diff
@@ -36,35 +36,35 @@ tests:
...
 <body>
   <header>
-    <h1>Training</h1>
+    <h1>Entrenamiento</h1>
     <nav>
       <ul>
-        <li><a href="#stealth">Stealth &amp; Agility</a></li>
-        <li><a href="#combat">Combat</a></li>
-        <li><a href="#weapons">Weapons</a></li>
+        <li><a href="#stealth">Sigilio & Agilidad</a></li>
```
Comment (hestellez) [inline curriculum/challenges/spanish/01-responsive-web-design/applied-accessibility/make-screen-reader-navigation-easier-with-the-footer-landmark.spanish.md]: You mean **sigilo** ?

File: curriculum/challenges/spanish/01-responsive-web-design/applied-accessibility/make-screen-reader-navigation-easier-with-the-footer-landmark.spanish.md [hunk 2]
```diff
@@ -36,35 +36,35 @@ tests:
...
+        <li><a href="#weapons">Armas</a></li>
       </ul>
     </nav>
   </header>
   <main>
     <section id="stealth">
-      <h2>Stealth &amp; Agility Training</h2>
-      <article><h3>Climb foliage quickly using a minimum spanning tree approach</h3></article>
-      <article><h3>No training is NP-complete without parkour</h3></article>
+      <h2>Entrenamiento de Agilidad y Sigilio</h2>
```
Comment (hestellez) [inline curriculum/challenges/spanish/01-responsive-web-design/applied-accessibility/make-screen-reader-navigation-easier-with-the-footer-landmark.spanish.md]: I think it's **sigilo** :)

File: curriculum/challenges/spanish/01-responsive-web-design/applied-accessibility/make-screen-reader-navigation-easier-with-the-footer-landmark.spanish.md [hunk 3]
```diff
@@ -36,35 +36,35 @@ tests:
...
-      <article><h3>Breadth-first or depth-first in multi-weapon training?</h3></article>
+      <h2>Entrenamiento de Armas</h2>
+      <article><h3>Espadas: la mejor herramienta para dividir y conquistar literalmente</h3></article>
+      <article><h3>¿Amplitud o profundidad primero en entrenamiento de armas múltiples?</h3></article>
     </section>
   </main>
 
 
-  <div>&copy; 2018 Camper Cat</div>
+  <div>2018 Camper Cat</div>
```
Comment (hestellez) [inline curriculum/challenges/spanish/01-responsive-web-design/applied-accessibility/make-screen-reader-navigation-easier-with-the-footer-landmark.spanish.md]: Don't forget **&copy;**

File: curriculum/challenges/spanish/01-responsive-web-design/applied-accessibility/make-screen-reader-navigation-easier-with-the-nav-landmark.spanish.md [hunk 4]
```diff
@@ -38,32 +38,31 @@ tests:
...
   <header>
-    <h1>Training with Camper Cat</h1>
-
+        <h1>Entrenamiento con Camper Cat</h1>
     <div>
       <ul>
-        <li><a href="#stealth">Stealth &amp; Agility</a></li>
-        <li><a href="#combat">Combat</a></li>
-        <li><a href="#weapons">Weapons</a></li>
+         <li><a href="#stealth">Sigilio & Agilidad</a></li>
```
Comment (hestellez) [inline curriculum/challenges/spanish/01-responsive-web-design/applied-accessibility/make-screen-reader-navigation-easier-with-the-nav-landmark.spanish.md]: Sigilo

File: curriculum/challenges/spanish/01-responsive-web-design/applied-accessibility/make-screen-reader-navigation-easier-with-the-nav-landmark.spanish.md [hunk 5]
```diff
@@ -38,32 +38,31 @@ tests:
...
       </ul>
     </div>
 
   </header>
   <main>
     <section id="stealth">
-      <h2>Stealth &amp; Agility Training</h2>
-      <article><h3>Climb foliage quickly using a minimum spanning tree approach</h3></article>
-      <article><h3>No training is NP-complete without parkour</h3></article>
+      <h2>Entrenamiento de Agilidad y Sigilio</h2>
```
Comment (hestellez) [inline curriculum/challenges/spanish/01-responsive-web-design/applied-accessibility/make-screen-reader-navigation-easier-with-the-nav-landmark.spanish.md]: Sigilo

File: curriculum/challenges/spanish/01-responsive-web-design/applied-accessibility/make-screen-reader-navigation-easier-with-the-nav-landmark.spanish.md [hunk 6]
```diff
@@ -38,32 +38,31 @@ tests:
...
+      <article><h3>Despacha enemigos múltiples con tácticas multiproceso</h3></article>
+      <article><h3>Adiós mundo: 5 maneras probadas de noquear a un oponente</h3></article>
     </section>
     <section id="weapons">
-      <h2>Weapons Training</h2>
-      <article><h3>Swords: the best tool to literally divide and conquer</h3></article>
-      <article><h3>Breadth-first or depth-first in multi-weapon training?</h3></article>
+      <h2>Entrenamiento de Armas</h2>
+      <article><h3>Espadas: la mejor herramienta para dividir y conquistar literalmente</h3></article>
+      <article><h3>¿Amplitud o profundidad primero en entrenamiento de armas múltiples?</h3></article>
```
Comment (hestellez) [inline curriculum/challenges/spanish/01-responsive-web-design/applied-accessibility/make-screen-reader-navigation-easier-with-the-nav-landmark.spanish.md]: You could omit the word _primero_

File: curriculum/challenges/spanish/01-responsive-web-design/applied-accessibility/make-screen-reader-navigation-easier-with-the-footer-landmark.spanish.md (hunk 1 above)
Comment (s-Hale) [reply curriculum/challenges/spanish/01-responsive-web-design/applied-accessibility/make-screen-reader-navigation-easier-with-the-footer-landmark.spanish.md]: Thanks! Not sure what happened there. I've corrected it.

File: curriculum/challenges/spanish/01-responsive-web-design/applied-accessibility/make-screen-reader-navigation-easier-with-the-footer-landmark.spanish.md (hunk 2 above)
Comment (s-Hale) [reply curriculum/challenges/spanish/01-responsive-web-design/applied-accessibility/make-screen-reader-navigation-easier-with-the-footer-landmark.spanish.md]: Corrected!

File: curriculum/challenges/spanish/01-responsive-web-design/applied-accessibility/make-screen-reader-navigation-easier-with-the-nav-landmark.spanish.md (hunk 5 above)
Comment (s-Hale) [reply curriculum/challenges/spanish/01-responsive-web-design/applied-accessibility/make-screen-reader-navigation-easier-with-the-nav-landmark.spanish.md]: Fixed.

**Review by blueshabat — APPROVED** [review]

**Review by blueshabat — APPROVED** [review]

**Comment by lynxlynxlynx** [comment]
Thanks!

**Pull request merged**